OSDOCS-4275: Updating future removal section for 4.13 removals (#51196)
openshift/openshift-docs: release_notes/ocp-4-12-release-notes.adoc

[id="ocp-4-12-future-removals"]
=== Future Kubernetes API removals

> The next minor release of {product-title} is expected to use Kubernetes 1.26. Currently, Kubernetes 1.26 is scheduled to remove a `v1beta1` and a `v2beta2` API.

Reviewer: ```suggestion
The next minor release of {product-title} is expected to use Kubernetes 1.26. Currently, there are a number of deprecated APIs scheduled to be removed in Kubernetes 1.26.
```
I still think the text: _"a `v1beta1` and a `v2beta2`"_ is odd. It tries to add some specifics, but they are useless without the full API name. Additionally, it is not consistent from release to release, so something more generic is less work in the long run.

Author: Bah, I made that fix on the other release notes PR I had (#51195), but forgot to do it here. Will remove this here too!
